go/vt/vtgate/planbuilder/physical/route_planning.go

```diff
@@ -161,12 +159,91 @@ func CreatePhysicalOperator(ctx *plancontext.PlanningContext, opTree abstract.Lo
 			return nil, vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "multi shard update with limit is not supported")
 		}
 
+		return r, nil
+	case *abstract.Delete:
+		var typ topodatapb.TabletType
+		var dest key.Destination
+		var ovq string
+
```

Comment:
nit: it would be nice to extract this case into a method. Preferably, the rest of the cases should be extracted too

Reply:
Fixed